crates/sdk/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+//! Blueprint SDK
+
+#[cfg(feature = "tangle")]
+pub use tangle_deps::*;
+#[cfg(feature = "tangle")]
+mod tangle_deps {
+    pub use tangle_blueprint_sdk::*;
+}
+
+#[cfg(feature = "testing")]
+pub use testing::*;
+#[cfg(feature = "testing")]
+/// Testing utilities and helpers
+mod testing {
+    /// Tangle-specific client for testing
+    pub use gadget_client_tangle;
+    /// Tangle-specific testing utilities
+    pub use gadget_tangle_testing_utils;
+    /// General testing utilities for blueprints
+    pub use gadget_testing_utils;
+    /// Temporary file and directory management for tests
+    pub use tempfile;
+}
+
+// Build utilities
+#[cfg(feature = "build")]
+pub use build::*;
+#[cfg(feature = "build")]
+/// Build-time utilities for blueprint compilation
+mod build {
+    /// Build utilities for blueprint compilation
+    pub use blueprint_build_utils;
+    /// Metadata generation for blueprints
+    pub use blueprint_metadata;
+}
+
+#[cfg(feature = "networking")]
+pub use networking::*;
+#[cfg(feature = "networking")]
+/// Networking utilities for blueprints
+mod networking {
+    /// Networking utilities for blueprints
+    pub use gadget_networking;
+}
```

Comment:
```suggestion
#[cfg(feature = "networking")]
/// Networking utilities for blueprints
pub use gadget_networking as networking;
```

Assuming there aren't anymore networking dependencies

Reply:
@drewstone if the only dependency being added with the `networking` feature is `gadget-networking`, then we can just re-export it directly, rather than create a whole new module. Just like what's being done with `gadget-config`, for example.

Reply:
I originally chose this format:

```rust
#[cfg(feature = "<feat>")]
pub use category::*;
#[cfg(feature = "<feat>")]
mod category {
    pub use element_in_category;
}
```

That way I don't have to add a single feature flag to every single export. Yes, there's currently only one dependency, but I don't know if that will be the case in the future, so I keep things consistent.

But, I do see your point in wanting to amend the syntax to be shorter; I will look into it.